servers/debugger/servers_debugger.cpp

```diff
@@ -342,10 +342,13 @@ class ServersDebugger::ServersProfiler : public EngineProfiler {
 		}
 		ServerInfo &srv = server_data[name];
 
-		ServerFunctionInfo fi;
-		fi.name = p_data[1];
-		fi.time = p_data[2];
-		srv.functions.push_back(fi);
+		int i = 1;
+		while (i < p_data.size() - 1) {
+			ServerFunctionInfo fi;
+			fi.name = p_data[i++];
+			fi.time = p_data[i++];
+			srv.functions.push_back(fi);
```

Comment:
This is valid, but we tend to make increments separate from using variables as this otherwise adds cognitive overload to triple check whether we're pre or post-incrementing.

IMO this would communicate the intent better:
```suggestion
			ServerFunctionInfo fi;
			fi.name = p_data[i];
			fi.time = p_data[i + 1];
			srv.functions.push_back(fi);
			i += 2;
```

Reply:
Or in this case even a `for` loop with `i += 2` increment.